content/recipes/router-module.md

```diff
@@ -24,6 +25,8 @@ In such a case, instead of repeating the `/dashboard` prefix within each control
 export class AppModule {}
 ```
 
+Next we set the `dashboard` route in the controller decorator like this `@Controller ('dashboard')`.
```

Comment:
I believe this should be `@Controller()` because the `RouterModule` will handle setting the path, right?

Reply:
Correct 👍

Reply:
So it could go:
 
Next we set the `dashboard` route in the `@Controller()` like this `@Controller ('dashboard')`. ?

Reply:
No, we shouldn't need to set the route at all, because the `RouterModule` already does it. The class just needs the `@Controller()` decorator, as is, to my knowledge. I'll need to play with it and verify that

Reply:
In fact, there is something strange

Reply:
This sentence was misleading & incorrect so I removed it.